packages/admin-ui-docs/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file is temporary, because `ts` complains about type declaration in some `js` components of docusaurus. Later I will try to eject the same componentes in TypeScript